documentation/book/proc-deploying-cluster-operator-kubernetes-to-watch-whole-cluster.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// assembly-cluster-operator.adoc
+// assembly-operators-cluster-operator.adoc
+
+[id='deploying-cluster-operator-kubernetes-to-watch-whole-cluster-{context}']
+= Deploying the Cluster Operator to watch the whole cluster
+
+Cluster Operator can be installed to manage Kafka, Kafka Connect and Kafka Mirror Maker in the whole {ProductPlatformName} cluster across all {Namespaces}.
```

Comment:
Should we mention that it will automatically pick up new namespaces?

Reply:
I added a sentence about this ... but you might want to review it ;-)